includes/functions.php

```diff
@@ -580,6 +580,7 @@ function createHost($host, $community = NULL, $snmpver, $port = 161, $transport
         'port' => $port,
         'transport' => $transport,
         'status' => '1',
+        'ip' => $ip,
```

Comment:
From the looks of it, $ip will be just the IP itself. This needs converting using the function we use elsewhere.